setup.cfg

```diff
@@ -54,6 +54,8 @@ filterwarnings =
   ignore:nodes.Node.traverse\(\) is obsoleted by Node.findall\(\)
   # Sphinx raises this for now
   ignore:'imghdr' is deprecated and slated for removal in Python 3.13:DeprecationWarning
+  ignore:Deprecated call to `pkg_resources.declare_namespace('sphinxcontrib')`:DeprecationWarning
+  ignore:The alias 'sphinx.util.SkipProgressMessage' is deprecated
```

Comment:
Adding this even if I can't get this to raise on the CI. This should prevent it happening when packaging this library.